src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -23,8 +23,8 @@ import cromwell.engine.{AbortRegistrationFunction, WorkflowDescriptor, _}
 import cromwell.logging.WorkflowLogger
 import cromwell.parser.BackendType
 import cromwell.util.StringUtil._
-import cromwell.util.TryUtil
-import cromwell.util.google.{GoogleCloudStorage, GoogleCloudStoragePath, GoogleCredentialFactory}
```

Comment:
No real reason, I just felt like the shorter version was easier to work with.  I'll change it back if people disagree